addons/web/static/src/js/control_panel/control_panel_model.js

```diff
@@ -549,23 +549,13 @@ odoo.define('web.ControlPanelModel', function (require) {
          * @private
          */
         _activateDefaultFilters() {
-            const defaultFilters = [];
-            const defaultFavorites = [];
-            for (const fId in this.state.filters) {
-                if (this.state.filters[fId].isDefault) {
-                    if (this.state.filters[fId].type === 'favorite') {
-                        defaultFavorites.push(this.state.filters[fId]);
-                    } else {
-                        defaultFilters.push(this.state.filters[fId]);
-                    }
-                }
-            }
-            if (this.activateDefaultFavorite && defaultFavorites.length) {
+            if (this.defaultFavoriteId) {
                 // Activate default favorite
-                this.toggleFilter(defaultFavorites[0].id);
+                this.toggleFilter(this.defaultFavoriteId);
             } else {
                 // Activate default filters
-                defaultFilters
+                Object.values(this.state.filters)
+                    .filter((f) => f.isDefault && f.type !== 'favorite')
```

Comment:
we are in else and only one favorite can be a default

Reply:
We can still have `search_disable_custom_filters` in the context preventing adding custom default favorites, right ?

Reply:
exact